.github/workflows/alpha-integration-tests.yaml

```diff
@@ -0,0 +1,93 @@
+name: DataRepo Alpha Integration Tests
+env:
+  SPRING_PROFILES_ACTIVE: google,cloudsql,alpha
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  GOOGLE_CLOUD_PROJECT: terra-datarepo-alpha
+  GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-alpha-data
+  IT_JADE_API_URL: https://data.alpha.envs-terra.bio
+  PGHOST: 127.0.0.1
+  PGPASSWORD: postgres
+  DB_DATAREPO_URI: "jdbc:postgresql://${PGHOST}:5432/datarepo"
+  DB_STAIRWAY_URI: "jdbc:postgresql://${PGHOST}:5432/stairway"
```

Comment:
After looking at the helm definitions, I think we could actually remove any env variable that's already defined there. 
* remove SPRING_PROFILES_ACTIVE, GOOGLE_CLOUD_PROJECT
* We could remove GOOGLE_CLOUD_DATA_PROJECT and reference [GOOGLE_SINGLEDATAPROJECTID](https://github.com/broadinstitute/terra-helmfile/pull/156/files#diff-9315f4b742c97ef06713007f74015e07L48) instead below. But, we're going to have to change how we reference data projects before "going live" so it might be good to have it defined here. 
* IT_JADE_API_URL could be added to helm & removed here, [like we're doing in some of the other environments](https://github.com/broadinstitute/datarepo-helm-definitions/blob/master/dev/dev/devDeployment.yaml#L33)

Reply:
1. `SPRING_PROFILES_ACTIVE` is required so that `@ActiveProfiles` actually uses that annotation and the `alpha` overrides
2. I may be able to remove `GOOGLE_CLOUD_PROJECT`, but I think it is needed for `gcloud` commands
3. `GOOGLE_CLOUD_DATA_PROJECT` is used for `get-iam-policy`
4. `IT_JADE_API_URL` needs to be defined locally on the GitHub Runner for integration tests in addition to remotely, otherwise tests will error.